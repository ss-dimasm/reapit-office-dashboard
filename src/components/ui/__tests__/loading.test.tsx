import { shallow } from 'enzyme'
import React from 'react'
import LoadingIndicator from '../loading'

const mockLoading = LoadingIndicator as jest.Mock<any>

jest.mock('../loading')

describe('<Loading/>', () => {
  beforeEach(() => {
    mockLoading.mockImplementation(() => ({
      label: 'Loading',
    }))
  })

  afterEach(() => {
    jest.clearAllMocks()
  })

  it('should render Loading Indicator Component', () => {
    const wrapper = shallow(<LoadingIndicator label={'Loading'} />)

    expect(wrapper).toMatchSnapshot()
  })
})
