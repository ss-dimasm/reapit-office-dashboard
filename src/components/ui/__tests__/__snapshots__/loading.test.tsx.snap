// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading/> should render Loading Indicator Component 1`] = `<Component />`;
